### PR TITLE
HOSTEDCP-1685: Use multiarch images for hypershift aws e2e

### PR DESCRIPTION
**Pull request by Patryk-Stefanski**

### CODE REVIEW

File: ci-operator/config/openshift/hypershift/openshift-hypershift-main.yaml [hunk 1]
```diff
@@ -79,6 +79,18 @@ releases:
...
+      architecture: multi
+      product: ocp
+      stream: nightly
+      version: "4.16"
+  multi-latest:
+    candidate:
+      architecture: multi
+      product: ocp
+      stream: nightly
+      version: "4.16"
```
Comment (sjenning) [inline ci-operator/config/openshift/hypershift/openshift-hypershift-main.yaml:93]: `initial` and `latest` are the same here

File: ci-operator/config/openshift/hypershift/openshift-hypershift-release-4.16.yaml [hunk 2]
```diff
@@ -80,6 +80,18 @@ releases:
...
+      architecture: multi
+      product: ocp
+      stream: nightly
+      version: "4.16"
+  multi-latest:
+    candidate:
+      architecture: multi
+      product: ocp
+      stream: nightly
+      version: "4.16"
```
Comment (sjenning) [inline ci-operator/config/openshift/hypershift/openshift-hypershift-release-4.16.yaml:94]: same here

File: ci-operator/config/openshift/hypershift/openshift-hypershift-release-4.17.yaml [hunk 3]
```diff
@@ -79,6 +79,18 @@ releases:
       include_built_images: true
       name: "4.17"
       namespace: ocp
+  multi-initial:
+    candidate:
+      architecture: multi
+      product: ocp
+      stream: nightly
+      version: "4.16"
```
Comment (sjenning) [inline ci-operator/config/openshift/hypershift/openshift-hypershift-release-4.17.yaml:87]: this is a y-stream upgrade and we typically test z-stream with same `version` and `relative: 1`

Comment (Patryk-Stefanski) [reply ci-operator/config/openshift/hypershift/openshift-hypershift-release-4.17.yaml:87]: yeah I was just copying what was already in there for the standard release and it seems like were using 4.16 > 4.17 for some reason.  Ill change it to do as suggested.

**Comment by sjenning** [comment]
/pj-rehearse pull-ci-openshift-hypershift-release-4.16-e2e-aws

**Comment by sjenning** [comment]
prow job timed out but no failures related to this PR

/lgtm
/pj-rehearse ack

**Pull request merged**